phpunit.xml.dist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         syntaxCheck="true">
+    <testsuites>
+        <testsuite name="unit">
+            <directory suffix="Test.php">./Tests</directory>
```

Comment:
it is a common practice to have symfony bundles not contain a src folder
but have the classes directly in root. adding a tests folder is then a
bit weird. the gitattributes are set up accordingly to exclude ./Tests

the name is the unit tests suite. you mean something like
"HttpClientBundle Unit Tests" ?